bsp/env/freedom-e300-hifive1/entry.S

```diff
@@ -1,55 +1,47 @@
-// See LICENSE for License Details
+// See LICENSE for license details
```

Comment:
Probably a common directory such as `bsp/env` (or perhaps one more appropriately named).  
The symlinking kept open the option of adding board-specific refinements later, but I'm no longer convinced that these are needed in the foreseeable future.